src/pybind/mgr/cephadm/module.py

```diff
@@ -1330,7 +1331,7 @@ def _get_container_image(self, daemon_name: str) -> Optional[str]:
             image = self.container_image_haproxy
         elif daemon_type == 'keepalived':
             image = self.container_image_keepalived
-        elif daemon_type == CustomContainerService.TYPE:
+        elif daemon_type == CustomContainerService.TYPE or daemon_type == 'ceph-exporter':
```

Comment:
is this also meant to be a temporary thing? Why would we not be able to figure out the image for this exporter? Unless it isn't meant to be containerized?